library-templates/generated-connector-template/files/ballerina/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+distribution = "{{BAL_VERSION}}"
+org = "ballerinax"
+name = "{{MODULE_NAME_CC}}"
+version = "{{MODULE_VERSION}}"
+license = ["Apache-2.0"]
+authors = ["Ballerina"]
+keywords = [] // TODO: Add keywords
```

Comment:
Can't we get the keywords as an input?

Reply:
Thought of keeping it this way to avoid over-complicating the workflow inputs. and IMO it makes more sense to keep these type of project-specific metadata (e.g., icons) as TODOs in the template, which suggest a more generic approach. wdyt?

Reply:
+1